src/components/header/header.module.scss

```diff
@@ -1,51 +1,26 @@
 @import '../../styles/variables.module.scss';
+@import '../../styles/fonts.module.scss';
 
 .header {
   position: relative;
-  margin-bottom: 2.5 * $space-xxl;
   pointer-events: none;
+  margin-top: 56px;
 
-  @media (max-width: $max-md) {
-    margin-bottom: $space-xxxl;
+  @media (min-width: $md) {
+    margin-top: 0;
```

Comment:
Removing the margin bottom from the page header, leaving it up to each page to decide this spacing. Top margin instead works, as all screens from all pages match with each other on that.